### ISSUE TITLE
"MUST ignore proprietary metadata properties" at the end of 3.3 in EPUB-RS 3.3

### ISSUE THREAD

**Issue by murata2makoto** [mattgarrish/epub-revision#2134]
This sentence appears at the end of [3.3](https://w3c.github.io/epub-specs/epub33/rs/#sec-pkg-doc-metadata) in EPUB-RS 3.3.

First, "proprietary metadata" is not defined in EPUB 3.3 or EPUB 3.3 RS.

Second, this sentence does not have a test.

Third, I have never understood what this sentence means (see #807)

**Comment by mattgarrish** [comment]
> Third, I have never understood what this sentence means

Apple used to define orientation-lock properties for fixed layouts, for example. I don't know that these conflicted with rendition:orientation, but what would you call this metadata if not proprietary? 3rd party? Vendor-specific?

The restriction, though, would be better placed on not _making_ proprietary metadata extensions that conflict with the behaviours, as if you make something like that why would you turn around and ignore it? I don't believe we were trying to ban fxl extensions as much as not fragment standardized behaviours.

In other words, I'd move the section under FXL and make it more like:

> Extension Properties
>
> Reading System developers MAY introduce additional properties to control the rendering of Fixed Layout Documents provided the expressions do not use the `rendition:` prefix and do not conflict behaviorally with those defined in this section.

And then add some boilerplate about how standardizing within W3C is preferred to ensure interop.

**Comment by iherman** [comment]
@murata2makoto on the question of missing tests, see my comment at https://github.com/w3c/epub-specs/issues/2132#issuecomment-1080169234.

One of the core tasks of the coming months will be to complete the test suite. It is a tedious work of adding tests to all statements, and any help will be necessary...

cc @dlazin

**Comment by murata2makoto** [comment]
@mattgarrish 

> Apple used to define orientation-lock properties for fixed layouts, for example. I don't know that these conflicted with rendition:orientation, but what would you call this metadata if not proprietary? 3rd party? Vendor-specific?

I was wondering if this statement is to warn Apple but I was not sure.  

This requirement was useful at that time, but do we still need it?  Your rewrite implicitly recommends proprietary extensions.

**Comment by mattgarrish** [comment]
> This requirement was useful at that time, but do we still need it?

Ya, I don't know how likely it is we'll see more now that the metadata is established, but to me there's no harm in telling people they can't use the rendition prefix space or conflict with the existing properties. Better to be clear than say nothing. Otherwise, someone could create their own rendition:* property and it would be perfectly valid.

> Your rewrite implicitly recommends proprietary extensions.

I wouldn't say it recommends any more than our allowing custom attributes encourages that practice, but that's also why I suggested we should say the usual things about standardizing within W3C rather than coming up with your own.

**Comment by mattgarrish** [comment]
We could also make this section parallel the custom attributes section to better harmonize the two. It states the attributes are for experimentation, for example, which is typically considered okay.

**Comment by iherman** [comment]
Isn't pr #2138 meant to fix this issue? If so, we should mark it as such...

**Comment by mattgarrish** [comment]
> Isn't pr #2138 meant to fix this issue?

Yes, went off to hunt down the RS diff/preview links to paste in and forgot to add a "fixes" statement. Done now.
